v3/example_test.go

```diff
@@ -20,5 +20,20 @@ func ExampleDecode() {
 	fmt.Println(ll.Lat, ll.Lon)
 
 	// Output:
-	// 35.63992106908978 139.7256515775034
+	// 35.63992106908978 139.72565157750344
+}
+
+func ExampleNeighbours() {
```

Comment:
according to godoc conventions, this needs to be `func ExamplePosition_Neighbours()`